ow_lander/src/ow_lander/actions.py

```diff
@@ -1367,3 +1377,33 @@ def execute_action(self, goal):
         self._set_succeeded("Reached commanded pan/tilt values")
       else:
         self._set_preempted("Action was preempted")
+
+
+class ActivateCommsServer(ActionServerBase):
+
+  name          = 'ActivateComms'
+  action_type   = ow_lander.msg.ActivateCommsAction
+  goal_type     = ow_lander.msg.ActivateCommsGoal
+  feedback_type = ow_lander.msg.ActivateCommsFeedback
+  result_type   = ow_lander.msg.ActivateCommsResult
+
+  def __init__(self):
+    super(ActivateCommsServer, self).__init__()
+    self._start_server()
+
+  def execute_action(self, goal):
+
+    # This action only draws a large amount of power from the battery for some
+    # period of time. There is no simulation of mission to Earth comms and
+    # uplinking and downlinking, for the purpose of this action, are treated the
+    # same and draw the same amount of power.
+
+    power_load_while_active = rospy.get_param('/ow_power_system/comms_active')
+    duration_of_activation = rospy.get_param('/ow_power_system/comms_duration')
```

Comment:
This may be a dumb question, but are these values both ROS parameters and read from a config file?  Perhaps I just haven't yet seen the code that connects them.